test/e2e/utils.py

```diff
@@ -13,16 +13,22 @@
 # limitations under the License.
 import time
 import logging
+import json
+import requests
+from urllib.parse import urlparse
 from kubernetes import client, config
 from kfserving import KFServingClient
 
 logging.basicConfig(level=logging.INFO)
 
 KFServing = KFServingClient(config_file="~/.kube/config")
+KFSERVING_NAMESPACE = "kfserving-system"
+KFSERVING_TEST_NAMESPACE = "kfserving-ci-e2e-test"
```

Comment:
You can leave it alone but i'm not sure is that better to rename as `KFSERVING_E2E_TEST_NAMESPACE`,  since Ellis mentioned that we will have e2e tests, performance test etc... if you think no need separate that, then I suggest we can change the value to `kfserving-ci-test`.